v3/plugins/joaompinto/asciidoctor-vscode/latest/meta.yaml

```diff
@@ -0,0 +1,12 @@
+publisher: joaompinto
+apiVersion: v2
+id: joaompinto.asciidoctor-vscode
```

Comment:
`id` shouldn't be included in the meta.yaml (`id` for this plugin would be `joaompinto/asciidoctor-vscode/latest`)

Reply:
Done. Thanks.